composer.json

```diff
@@ -42,12 +42,12 @@
     }
   ],
   "require-dev": {
-    "symfony/framework-bundle": "^2.3||^3.0||^4.0",
+    "symfony/framework-bundle": "^2.3||^3.0||^4.0||^5.0",
     "monolog/monolog": "~1.8",
-    "symfony/proxy-manager-bridge": "^2.3||^3.0||^4.0",
-    "phpunit/phpunit": "^4.8||^5.0",
-    "matthiasnoback/symfony-config-test": "^2.0",
-    "matthiasnoback/symfony-dependency-injection-test": "^1.1",
+    "symfony/proxy-manager-bridge": "^2.3||^3.0||^4.0||^5.0",
+    "phpunit/phpunit": "^4.8||^5.0||^6.0||^7.0||^8.0",
```

Comment:
You should make this requirement more strict. You're subclassing a namespaced `TestCase` in `test/TestCase`, yet only PHPUnit 6.0+ has those (see [PHPUnit 6 release notes](https://phpunit.de/announcements/phpunit-6.html))

Reply:
There's also another option: make the class definitions conditional, e.g.
```php
namespace Broadway\Bundle\BroadwayBundle;

if (class_exists('\\PHPUnit_Framework_TestCase')) {
  abstract class TestCase extends \PHPUnit_Framework_TestCase
  {
  }
} elseif (class_exists('\\PHPUnit\\Framework\\TestCase')) {
  abstract class TestCase extends \PHPUnit\Framework\TestCase
  {
  }
}
```

Reply:
Yes, I know that solution, but I don't like it so much.
If for you is good enough your solution is not a problem for me to implement it :) I don't have a really strong opinion about it

Reply:
We should ask the maintainers what's their preferred way :)

Reply:
Yeah, is the right thing to do right now